public/input_text_load_data.js

```diff
@@ -119,6 +146,43 @@ function loadData(collection) {
   });
 }
 
+// Changes XML to JSON
```

Comment:
Yikes! Why do you need to translate XML to JSON? This can't be done losslessly. What are you trying to accomplish here?